CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [6.20.0] - 2023-11-15
+### Changed
+- Changed `k8s` API to work with provider 2.x
+- Changed Terraform `map` to `tomap` to make it work with newer TF provider.
```

Comment:
please mention which version of TF provider we are making this work with. 

Also, needs a major version update - 7.0.0

Reply:
done